src/virtual-select.js

```diff
@@ -1556,14 +1556,30 @@ export class VirtualSelect {
         selectedValuesCount += 1;
 
         if (showValueAsTags) {
-          const valueTooltipForTags = this.getTooltipAttrText(label, !(Utils.willTextOverflow($valueText.parentElement, label)), true);  
 
-          const valueTagHtml = `<span class="vscomp-value-tag" data-index="${d.index}" ${valueTooltipForTags}>
-              <span class="vscomp-value-tag-content">${label}</span>
-              <span class="vscomp-value-tag-clear-button">
-                <i class="vscomp-clear-icon"></i>
-              </span>
-            </span>`;
+            var valueTagHtml = '';
+            
+            //Will cause text overflow in runtime and if so, the tooltip information is prepared 
+            if(Utils.willTextOverflow($valueText.parentElement, label))
+            {
+              const valueTooltipForTags = this.getTooltipAttrText(label, false,  true);  
```

Comment:
I think this can be defined as empty, and inside the if it will have the expected value, then assigned into the valueTagHtml string, if it's empty it will add nothing, otherwise it will add the expected attribute. This way the if/else can be removed ;)

Reply:
@joselrio Both changes are now committed. Thank you for the great review 🚀